frontend/src/components/ProjectCardGrid/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect } from 'react'
+import useSWRInfinite from 'swr/infinite'
+import { useInView } from 'react-intersection-observer'
+import { Filter } from 'meilisearch'
+
+import { meiliIndex } from '@utils/meili'
+import ProjectCard, { IntersectionObserverRef } from '../ProjectCard'
+import styles from './index.module.scss'
+
+export const cardsPerRow = 3
+const defaultLimit = cardsPerRow * 6
```

Comment:
1. `cardsPerRow` is exported but not used anywhere?
2. How does this behave when there are less than 3 cards per row, e.g. on mobile?

Reply:
I fixed the export. 

We load chunks of 16 projects before we reach the limit.
`cardsPerRow` is used for calculating when we should load more projects. It loads more projects when the viewport reaches multiples of 12: (12, 24, 36, ...) regardless of the number of projects in a row.

Reply:
Huh? So `cardsPerRow` is used to calculate when to load projects but it loads new projects every 12 regardless of the actual cards per row?

Reply:
Yes

Reply:
Ok, can you refactor that to something that's a bit clearer please? I don't think make sense to name this `cardsPerRow`. Maybe `export const pageSize = 12` or something like that, and do away with the calculation?

Reply:
Resolved in f592c2a.